src/cdn.rs

```diff
@@ -123,7 +123,17 @@ pub(crate) fn invalidate_crate(config: &Config, cdn: &CdnBackend, name: &str) ->
             distribution_id,
             &[&format!("/{}*", name), &format!("/crate/{}*", name)],
         )
-        .context("error creating CDN invalidation")?;
+        .context("error creating web CDN invalidation")?;
+    }
+    if let Some(distribution_id) = config.cloudfront_distribution_id_static.as_ref() {
+        cdn.create_invalidation(
+            distribution_id,
+            &[
+                &format!("/rustdoc/{}*", name),
+                &format!("/sources/{}*", name),
```

Comment:
the question if this invalidation is too broad is still unanswered because the cloudfront docs are not clear on this. 

Since this is how we invalidate now I would leave it for this PR and dig into that point later.